examples/exotica_examples/launch/PythonPlanIKWithJntSmooth.launch

```diff
@@ -7,13 +7,18 @@
   <param name="robot_description" textfile="$(find exotica_examples)/resources/robots/lwr_simplified.urdf" />
   <param name="robot_description_semantic" textfile="$(find exotica_examples)/resources/robots/lwr_simplified.srdf" />
 
-  <!-- joy node -->
-  <node respawn="true" pkg="joy"
-        type="joy_node" name="joy_node" >
-    <param name="dev" type="string" value="/dev/input/js0" />
-    <param name="deadzone" value="0.12" />
-    <param name="autorepeat_rate" value="120.0"/>
+  <node pkg="keyboard" name="keyboard_node" type="keyboard_node.py">
+    <param name="hz" type="int" value="120"/>
   </node>
+  
+
+  <!-- joy node -->
```

Comment:
Use only one start and end to comment out. Easier to switch than if every line individually.

Reply:
I'm not sure what you mean?